lms/djangoapps/verify_student/services.py

```diff
@@ -75,7 +75,8 @@ def verifications_for_user(cls, user):
         Return a list of all verifications associated with the given user.
         """
         verifications = []
-        for verification in chain(SoftwareSecurePhotoVerification.objects.filter(user=user).order_by('-created_at'),
+        for verification in chain(VerificationAttempt.objects.filter(user=user).order_by('-created'),
```

Comment:
Should this be 'created_at'? Does it matter if it's not matching others?

Reply:
This is due to the old models not inheriting from `TimestampedModel`